etest/progressbar.py

```diff
@@ -0,0 +1,46 @@
+"""Progress bars."""
```

Comment:
We should move progress bars to a follow up pull request completely to keep these easier to review.  This is a completely new feature on top of etest-build.

Reply:
I wanted to land the complete command, but it does make sense to stash these for a following PR